components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -917,6 +920,40 @@ public String getPassiveSTSWReply() {
         return passiveSTSWReply;
     }
 
+    /**
+     * Returns the passive sts wreply logout url.
+     *
+     * @return Passive sts wreply logout url.
+     */
+    public String getPassiveSTSWReplyLogout() {
+
+        if (passiveSTSWReplyLogout != null) {
+            return passiveSTSWReplyLogout;
+        }
+
+        InboundAuthenticationRequestConfig[] authRequest = serviceProvider
+                .getInboundAuthenticationConfig().getInboundAuthenticationRequestConfigs();
+
+        if (authRequest != null) {
+            for (int i = 0; i < authRequest.length; i++) {
+                if ("passivests".equalsIgnoreCase(authRequest[i].getInboundAuthType())) {
```

Comment:
Let's move this to a constant.

Reply:
Fixed with [ab85885](https://github.com/wso2/carbon-identity-framework/pull/5129/commits/ab8588512be9c241a44f76890e44d705a568b6ef)